Userland/Libraries/LibWeb/Selection/Selection.cpp

```diff
@@ -493,12 +493,14 @@ void Selection::move_offset_to_next_character(bool collapse_selection)
         return;
 
     auto& text_node = static_cast<DOM::Text&>(*anchor_node);
-    if (auto offset = text_node.grapheme_segmenter().next_boundary(focus_offset()); offset.has_value()) {
+    auto focus_offset_in_bytes = Utf8View { text_node.data() }.byte_offset_of(focus_offset());
```

Comment:
Since this is more of a lateral step in terms of correctness - while we're here, can we slap a FIXME comment on these?